main.py

```diff
@@ -0,0 +1,51 @@
+import random
+import sys
+
+user_name = input("Bitte geben Sie Ihren Benutzernamen ein: ")
+print(f"Hallo {user_name}! Ich denke an eine Zahl zwischen 1 und 20. Welche Zahl ist es?")
+
+def try_input(amount_of_tries):
+    while True:
+            try: 
+                if amount_of_tries == 1:
+                    guess = int(input("Bitte geben Sie eine Zahl zwischen 1 und 20 ein: "))
+                else: 
+                    guess = int(input())
+
+                if (guess < 1 or guess > 20):
```

Comment:
```suggestion
                if 1 > guess > 20:
```

This _should_ work, but test it :D

Reply:
Doesn't work unfortunately